app/clients/zendesk_sell.py

```diff
@@ -29,56 +35,47 @@ def _generate_lead_data(self) -> Dict[str, Union[str, List[str], Dict]]:
-                               f'{self.contact.main_use_case_details}',
-                'tags': [self.contact.support_type, self.contact.language],
+                'organization_name': contact.department_org_name,
+                'email': contact.email_address,
+                'description': f'Program: {contact.program_service_name}\n{contact.main_use_case}: '
+                               f'{contact.main_use_case_details}',
+                'tags': [contact.support_type, contact.language],
                 'status': 'New',
                 'custom_fields': {
                     'Product': ['Notify'],
                     'Source': 'Demo request form',
-                    'Intended recipients': recipients[self.contact.intended_recipients]
-                    if self.contact.intended_recipients in recipients else 'No value'
+                    'Intended recipients': recipients[contact.intended_recipients]
+                    if contact.intended_recipients in recipients else 'No value'
                 }
             }
         }
 
-    def __init__(self, contact: ContactRequest):
-        self.api_url = current_app.config['ZEN_DESK_SELL_API_URL']
-        self.token = current_app.config['ZEN_DESK_SELL_API_KEY']
-        self.contact = contact
-
-    def send_contact_request(self) -> int:
-        # FIXME: as a POC we will only send leads when a user requests a demo
-        # FIXME: remove this condition when we want to use zen desk sell fully for all inquiries
+    def send_contact_request(self, contact: ContactRequest) -> int:
         ret = 200
-        if 'demo' in self.contact.support_type.lower():
-            ret = self.send_lead()
+        if contact.is_demo_request():
+            ret = self.send_lead(contact)
 
         return ret
 
-    def send_lead(self) -> int:
+    def send_lead(self, contact: ContactRequest) -> int:
         # name is mandatory for zen desk sell API
-        assert len(self.contact.name), 'Zendesk sell requires a name for its API'
+        assert len(contact.name), 'Zendesk sell requires a name for its API'
 
         try:
             if not self.api_url or not self.token:
                 raise NotImplementedError
 
-            payload = self._generate_lead_data()
-            headers = {'Accept': 'application/json', 'Content-Type': 'application/json'}
-
             response = requests.post(
-                url=f'{self.api_url}/v2/leads/upsert?email={self.contact.email_address}',
-                data=json.dumps(payload),
-                headers=headers,
+                url=urljoin(self.api_url, f'/v2/leads/upsert?email={contact.email_address}'),
+                data=json.dumps(ZenDeskSell._generate_lead_data(contact)),
+                headers={'Accept': 'application/json', 'Content-Type': 'application/json'},
                 auth=BearerAuth(token=self.token),
+                timeout=30.0
```

Comment:
30s seems a long time, WDYT think about 5?

Reply:
will make the change